httomo/data/dataset_store.py

```diff
@@ -340,6 +340,10 @@ def aux_data(self) -> AuxiliaryData:
     def dtype(self) -> np.dtype:
         return self._data.dtype
 
+    @property
+    def has_gpu_flats(self) -> bool:
+        return self._data.has_gpu_flats
+    
```

Comment:
Is this property needed anymore with the latest changes from `feature/transparent-file-store`, it doesn't seem to be used anywhere? The PR description mentions that this was a property added to `DataSet`, but now that `DataSet` is gone, does the store need it anymore?

Reply:
good point, even mypy marked it...